projects/wmf_wikivoyage.yaml

```diff
@@ -0,0 +1,201 @@
+                - path: v1/mathoid.yaml
+                  options: '{{options.mathoid}}'
+            /data:
+              x-modules:
+                - path: v1/citoid.js
+                  options: '{{options.citoid}}'
+                - path: v1/lists.js
+                  options: '{{options.lists}}'
+                - path: v1/recommend.yaml
+                  options: '{{options.recommendation}}'
+                - path: v1/css.yaml
+                  options:
+                    host: '{{options.mobileapps.host}}'
+                - path: v1/javascript.yaml
+                  options:
+                    host: '{{options.mobileapps.host}}'
+        options: '{{options}}'
+
+  /{api:sys}:
+    x-modules:
+      - spec:
+          paths:
+            /table: &sys_table
+              x-modules:
+                - path: sys/table.js
+                  options:
+                    conf: '{{options.table}}'
+            /parsoid_bucket:
+              x-modules:
+                - path: sys/multi_content_bucket.js
+                  options:
+                    grace_ttl: '{{default(options.parsoid.grace_ttl, 86400)}}'
+                    index_ttl: '{{default(options.parsoid.index_ttl, 864000)}}'
+                    renew_expiring: true
+                    delete_probability: '{{default(options.parsoid.delete_probability, 1)}}'
+                    table_name_prefix: parsoid_ng
+                    main_content_type:
+                      name: html
+                      value_type: blob
+                    dependent_content_types:
+                      - name: data-parsoid
+                        value_type: json
+                      - name: section-offsets
+                        value_type: json
+            /key_value: &sys_key_value
+              x-modules:
+                - path: sys/key_value.js
+            /key_rev_value:
+              x-modules:
+                - path: sys/key_rev_value.js
+            /page_revisions:
+              x-modules:
+                - path: sys/page_revisions.js
+            /post_data: &sys_post_data
+              x-modules:
+                - path: sys/post_data.js
+            /action:
+              x-modules:
+                - path: sys/action.js
+                  options: "{{options.action}}"
+            /page_save:
+              x-modules:
+                - path: sys/page_save.js
+            /parsoid:
+              x-modules:
+                - path: sys/parsoid.js
+                  options: &parsoid_options
+                    parsoidHost: '{{options.parsoid.host}}'
+                    response_cache_control: '{{options.purged_cache_control}}'
+                    grace_ttl: '{{default(options.parsoid.grace_ttl, 86400)}}'
+            /mobileapps:
+              x-modules:
+                - path: sys/mobileapps.js
+                  options: '{{merge({"response_cache_control": options.purged_cache_control}, options.mobileapps)}}'
+            /mobile_bucket:
```

Comment:
Do we need the mobile buckets though?

Reply:
yes. the `no-store` header doesn't mean requests to the bucket are not made, they're just no-op